packages/slate-react/src/components/leaf.js

```diff
@@ -86,7 +90,11 @@ class Leaf extends React.Component {
       index,
     })
 
-    return <span data-offset-key={offsetKey}>{this.renderMarks()}</span>
+    return (
+      <span ref={this.setRef} data-offset-key={offsetKey}>
```

Comment:
Why this `ref=`?

Reply:
I forgot to remove it. I will remove that.